src/olympia/addons/tests/test_tasks.py

```diff
@@ -315,3 +315,82 @@ def test_basic(self):
 
         self.call_signing_mock.assert_called_with(current_file)
         assert current_file.cert_serial_num == 'abcdefg1234'
+
+
+# migrate_addons_that_require_sensitive_data_access
+class TestMigrateAddonsThatRequireSensitiveDataAccess(TestCase):
+
+    def setUp(self):
+        self.user = user_factory(
+            id=settings.TASK_USER_ID, username='taskuser',
+            email='taskuser@mozilla.com')
+
+        # Commented out until test fixes are merged
+        #create_default_webext_appversion()
+        # Inlined until test fixes are merged
+        from olympia.applications.models import AppVersion
+        AppVersion.objects.get_or_create(
+            application=amo.THUNDERBIRD.id, version='60.0')  # Default Min Version
+        AppVersion.objects.get_or_create(
+            application=amo.THUNDERBIRD.id, version='102.0') # Targeted Max Version
+        AppVersion.objects.get_or_create(
+            application=amo.THUNDERBIRD.id, version='*')
+
+
+    def test_addon_with_sda(self):
+        """Test addon with sensitive permissions. This should cause both the Admin and AdminReviewer Flags to be True"""
```

Comment:
The test is checking for `requires_sensitive_data_access` and `needs_sensitive_data_access_review`, but not any admin flags. Is this just a wording issue here, or a real code logic issue? Sensitive data does not need to escalate to admin review, just disable auto approval.